lib/piwik/test.js

```diff
@@ -119,6 +120,119 @@ describe('Piwik', function(){
         analytics.track('Completed Order', { total: 20 });
         analytics.called(window._paq.push, ['trackGoal', 10, 20]);
       });
+
+      it('should send one custom variable', function(){
+        analytics.track('event', { prop: true }, { integrations: { Piwik: { customVars: { 1: ["UserId","6116"] } } }});
+        analytics.called(window._paq.push, ['setCustomVariable', '1', 'UserId', '6116', 'page']);
+      });
+
+      it('should send multiple custom variables that get passed as cvar', function(){
+        analytics.track('event',
+          { prop: true },
+          { integrations: {
+              Piwik: {
+                cvar: {
+                  1: ["UserId","6116"],
+                  2: ['SubscriptionId', ''],
+                  3: ['PlanName', 'ENTERPRISE']
+                }
+              }
+            }
+          }
+        );
+        analytics.called(window._paq.push, ['setCustomVariable', '1', 'UserId', '6116', 'page']);
+        analytics.called(window._paq.push, ['setCustomVariable', '2', 'SubscriptionId', '', 'page']);
+        analytics.called(window._paq.push, ['setCustomVariable', '3', 'PlanName', 'ENTERPRISE', 'page']);
+      });
+
+      it('should send multiple custom variables that get passed as an object', function(){
```

Comment:
language on this is outdated now—should mirror above, `should send multiple custom variables that get passed as customVars`. You could also break the object you're passing into a variable to make later easier